webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -112,6 +112,22 @@ def new_mocked_get(**options):
             'summary': 'Some summary'
         }])
 
+    @mock.patch('requests.Session')
+    def test_bugzilla_api_bad_status_code(self, rsession):
+        model = models.BugzillaBugInfo
+
+        api = model()
+
+        def mocked_get(url, **options):
+            return Response("I'm a teapot", status_code=418)
+
+        rsession().get.side_effect = mocked_get
+        assert_raises(
+            models.BugzillaRestHTTPUnexpectedError,
+            api.get,
+            '123456789'
```

Comment:
You need a parameter when you call [`.get`](https://github.com/peterbe/socorro/blob/7ea020f29aa0694ff6b8749119b64e6728432b74/webapp-django/crashstats/crashstats/models.py#L969)